miner/worker.go

```diff
@@ -288,6 +288,29 @@ func (w *worker) close() {
 	close(w.exitCh)
 }
 
+// recalcRecommit recalculates the resubmitting interval upon feedback.
+// @target is the dration vale (time.Duration as floating point) where we want to go.
```

Comment:
Please don't introduce a new variable notation. There' is no `@` notation in Go. I'm also not sure "where we want to go" provides more info that not adding anything.

Reply:
OK -  make sense. 
I will removed the second line. Re `@` - sometimes I find it useful to quickly point to a parameter (consider: "@param" vs "parameter param".